plotly/tools.py

```diff
@@ -5108,7 +5111,9 @@ def make_kde(self):
             self.curve_y[index] = (scipy.stats.gaussian_kde
                                    (self.hist_data[index])
                                    (self.curve_x[index]))
-            self.curve_y[index] *= self.bin_size[index]
+
+            if self.histnorm == 'probability':
```

Comment:
Can you pull these into constants at the top of the file? Hard-coded things like this always raise 🚨's for me.